parm/config/config.ocnanalprep

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -x
```

Comment:
Other config files aren't turning off trace on their own, so this one shouldn't either.